awesomemotive/file-upload-types
# Remove unnecessary code that loads languages

Fixes #50 

Testing steps:
- install plugin
- go to wp-admin > settings > general and change language to russian
- go to wp-admin > settings > file upload types and see if the UI is in russian
- if not, go to wp-admin/update-core.php url and at the bottom of tyhe screen click button "Update translations"
- switch language to English and then again  russian and check if the UI is in russian
